src/models/rating.ts

```diff
@@ -0,0 +1,24 @@
+import mongoose, { Schema } from "mongoose";
+const systemRating = mongoose.model(
+    "systemRating",
+    new Schema({
+      name: {
+        type: String,
+        required: true,
+        unique: true,
+      },
+      grade: {
+        type:[Number],
+        required: true,
+      },
+      description: {
+        type:[String],
```

Comment:
This field is not an array of strings, just a `string`, am I right?
If so, please remove the brackets on the definition otherwise feel free to resolve